src/globals.ts

```diff
@@ -241,7 +257,15 @@ export const TracerGlobals = (() => {
     maxSizeForRequest = null,
     maxSizeForRequestOnError = null,
     lambdaTimeout = MAX_TRACER_ADDED_DURATION_ALLOWED,
-  }: TracerOptions) =>
+    maxSizeForStoredSpansInMemory = DEFAULT_MAX_SIZE_FOR_SPANS_SIZE_STORED_IN_MEMORY,
+  }: TracerOptions) => {
+    const parsedMaxSizeForRequestOnError =
+      maxSizeForRequestOnError ||
+      (process.env['LUMIGO_MAX_SIZE_FOR_REQUEST_ON_ERROR']
+        ? parseInt(process.env.LUMIGO_MAX_SIZE_FOR_REQUEST_ON_ERROR)
```

Comment:
Do we (want to) handle a `NaN` case for this value?

Reply:
I don't want to change the logic of this env var, any case I think we should remove it when we have better support for large invocations